Userland/Utilities/lsusb.cpp

```diff
@@ -18,15 +19,58 @@
 #include <stdio.h>
 #include <unistd.h>
 
+// FIXME: Hmmm, perhaps this should be part of the USBDB library?
+static const StringView s_device_classes[17] = {
+    "Use class information in the Interface Descriptors",
+    "Audio",
+    "Communications and CDC Control",
+    "HID (Human Interface Device)",
+    "Physical",
+    "Image",
+    "Printer",
+    "Mass Storage",
+    "Hub",
+    "CDC-Data",
+    "Smart Card",
+    "Content Security",
+    "Video",
+    "Personal Healthcare",
+    "Audio/Video Devices",
+    "Billboard Device Class",
+    "USB Type-C Bridge Class",
+};
+
+static StringView get_class_code(u32 class_code)
```

Comment:
`class_code_id` might be a better name, similar to `StringView Database::get_class(u8 class_id)`